README.md

```diff
@@ -116,6 +116,9 @@ The following parameters should be set in the `driver_config` for the individual
 
 The following parameters should be set in the main `driver_config` section as they are common to all platforms:
  - `vcenter_disable_ssl_verify` - Whether or not to disable SSL verification checks. Good when using self signed certificates. Default: false
+ - `vm_wait_timeout` - How many seconds to wait for VM connectivity. Default: 90
+ - `vm_wait_interval` - Check interval between tries on VM connectivity. Default: 2.0
+ - `vm_rollback` - Whether to roll back (destroy) VMs which failed VM connectivity checking. Default: false
```

Comment:
```suggestion
 - `vm_rollback` - Decision to roll back (destroy) VMs failing the VM connectivity check. Default: false
```

Reply:
I'd propose "Automatic roll back (destroy) of VMs failing the connectivity check. Default: false"? Decision sounds a bit weird to me. Non-native speaker though ;)